src/Consolonia.Gallery/Gallery/GalleryViews/SomeDialogWindow.axaml.cs

```diff
@@ -17,6 +17,7 @@ public SomeDialogWindow(double width, double height)
             Title = DialogTitle;
             Width = width;
             Height = height;
+            AnimateWindow = string.IsNullOrEmpty(Environment.GetEnvironmentVariable("CONSOLONIA_TEST"));
```

Comment:
We probably do not need environment variable if it's set in the code anyway any is kept inside application memory all the time. Simple static field would work?